src/player.cpp

```diff
@@ -11312,6 +11312,23 @@ void player::try_to_sleep()
                  _("It's hard to get to sleep on this %s."),
                  terlist[ter_at_pos].name.c_str() );
     }
+    // Deactivate active items, bionics and mutations
+    for ( auto &it : inv_dump() ) {
+        if ( it->active ) {
+            use( get_item_position(it) );
```

Comment:
Several active items give prompts when used (radio), having to go through them all can get quite annoying.

What if I want my RM13 combat armor to stay on because it gives the RAD_PROOF effect?

Have you tried it while carrying an active lightstrip?